content/cpp/concepts/math-functions/terms/atanh/atanh.md

```diff
@@ -0,0 +1,103 @@
+---
+Title: 'atanh()'
+Description: 'Computes the arc hyperbolic tangent(inverse hyperbolic tangent) of a number in radians.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`atanh()`** function in C++ returns the arc hyperbolic tangent (inverse hyperbolic tangent) of a number in `radians`.The area hyperbolic tangent is the inverse operation of the hyperbolic tangent.
+
+## Syntax
+
+```pseudo
+atanh(data_type x)
+```
+
+`x`: The value of mandatory argument in range `[-1,1]`.
+
+## Parameter
```

Comment:
Remove the parameter header and place this sentence at the end of the "Syntax" section.